python/tests/sql_aggregates/test_average.py

```diff
@@ -0,0 +1,183 @@
+import unittest
+from feldera import PipelineBuilder, Pipeline
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data = [{"insert":{"id": 0, "c1": 1, "c2": 2, "c3": 3, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},]       
+        return super().setUp()
+      
+    def execute_query(self, pipeline_name, expected_data, table_name, view_query):
+        sql = f'''CREATE TABLE {table_name}(
```

Comment:
there are many more data types that are interesting for avg, like double or decimal
can we have tests with these?

Reply:
My angle here was to test the aggregates with just the int types. That's why none of the tests include other types.

Reply:
that's fine, we'll do that next